packages/os/os.spec

```diff
@@ -97,6 +98,11 @@ Summary: Setting generator for templated settings values.
 %description -n %{_cross_os}schnauzer
 %{summary}.
 
+%package -n %{_cross_os}shibaken
+Summary: Setting generator for IMDS keys to host-containers.admin.user-data.
```

Comment:
nit: it's somewhat inaccurate to mention the specific setting here; the model could default to using the generator on a different key.